include/cantera/kinetics/Arrhenius.h

```diff
@@ -262,6 +263,12 @@ class TwoTempPlasmaRate final : public ArrheniusBase
     virtual void getParameters(AnyMap& node) const override;
 
     double evalFromStruct(const TwoTempPlasmaData& shared_data) {
+        if (m_Ea_R == NAN) {
+            m_Ea_R = 0.0;
+        }
```

Comment:
Any comparison (`==`, `!=`, `<`, `>` etc) always returns false for `NAN`. You need to use `isnan(m_Ea_R)`.

But I think this is the wrong place to be making this update -- you want to do this check when setting up the reaction, so in the constructor or `setParameters` function, not every time the rate is evaluated.